dtfs-central-api/src/v1/controllers/tfm/amendments/tfm-get-amendments.controller.js

```diff
@@ -82,21 +82,21 @@ const findAmendmentById = async (facilityId, amendmentId) => {
   try {
     const collection = await db.getCollection('tfm-facilities');
     const amendment = await collection
-      .aggregate([ // TODO SR-8
-        { $match: { _id: ObjectId(facilityId), 'amendments.amendmentId': ObjectId(amendmentId) } },
+      .aggregate([
+        { $match: { _id: { $eq: ObjectId(facilityId) }, 'amendments.amendmentId': { $eq: ObjectId(amendmentId) } } },
         {
           $addFields: {
             'amendments.ukefFacilityId': '$facilitySnapshot.ukefFacilityId',
           },
         },
         {
           $project: {
-            _id: 0,
+            _id: false, // TODO SR-8 Changed values in project to true or false to better represent the functionality
```

Comment:
Remove todo once noted:
```suggestion
            _id: false,
```